CHANGELOG.md

```diff
@@ -4,6 +4,36 @@
 All notable changes to this project will be documented in this file.
 <!--- END HEADER -->
 
+## [0.2.2](https://github.com/operations-site/compare/v0.2.1...v0.2.2) (2022-08-31)
```

Comment:
Not very important and doesn't need changing in this PR, but a little confusing to still be using the old 'operations-site' link. Might be worth updating the name of the repo locally and its remotes.